.github/workflows/publish-to-npm.yaml

```diff
@@ -1,22 +1,23 @@
 name: 'publish to npm'
 
 on:
-  workflow_dispatch:
-  release:
-    types: [published]
+    workflow_dispatch:
```

Comment:
@canhtrinh can we keep the indent the same? 2 space is what we use in other yaml files